content/en/docs/tutorials/cluster-management/kubelet-standalone.md

```diff
@@ -0,0 +1,521 @@
+
+Configure:
+
+{{< note >}}
+This tutorial uses the `10.241.1.0/24` network range (which is a private network).
+Make sure you don't have any conflicts with an existing network. If necessary,
+use a network that is available to you.
+{{< /note >}}
+
+```shell
+sudo mkdir -p /etc/kubernetes/manifests
+```
+
+```shell
+sudo tee /etc/kubernetes/kubelet.yaml <<EOF
+apiVersion: kubelet.config.k8s.io/v1beta1
+kind: KubeletConfiguration
+authentication:
+  webhook:
+    enabled: false
+authorization:
+  mode: AlwaysAllow
+enableServer: false
+logging:
+  format: text
+  sanitization: false
+readOnlyPort: 10250
+podCIDR: 10.241.1.0/24
+staticPodPath: /etc/kubernetes/manifests
+EOF
+```
+
+Install:
+
+```shell
+chmod +x kubelet
+sudo cp kubelet /usr/bin/
+```
+
+```shell
+sudo tee /etc/systemd/system/kubelet.service <<EOF
+[Unit]
+Description=Kubelet
+
+[Service]
+ExecStart=/usr/bin/kubelet \
+ --config=/etc/kubernetes/kubelet.yaml
+Restart=always
+
+[Install]
+WantedBy=multi-user.target
+EOF
+```
+
+```shell
+sudo systemctl daemon reload
+sudo systemctl enable --now kubelet.service
+```
+
+Quick test:
+
+```shell
+systemctl is-active kubelet.service
+```
+
+The output is similar to:
+
+```
+active
+```
+
+Detailed service check:
+
+```shell
+sudo journalctl -u kubelet.service
+```
+
+## Run a static Pod in the Kubelet
+
+In standalone mode, you can only run [static Pods](/docs/concepts/workloads/pods/#static-pods).
+The kubelet can fetch the static Pod configuration from a local filesystem,
+or over a network using HTTP.
+
+Create a manifest for a static Pod:
+
+```shell
+cat <<EOF > static-web.yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: static-web
+  labels:
+    role: myrole
+spec:
+  containers:
+    - name: web
+      image: nginx
+      ports:
+        - name: web
+          containerPort: 80
+          protocol: TCP
+EOF
+```
+
+Deploy:
+
+Copy the `static-web.yaml` manifest file to the `/etc/kubernetes/manifests` directory.
+
+```shell
+sudo cp static-web.yaml /etc/kubernetes/manifests/
+```
+
+### Find out information about the Kubelet and the static Pod
+
+The Pod networking plugin creates a pair of `veth` interfaces for each Pod
+(one of the pair is inside the newly made Pod, and the other is at the host level).
+
+Retrieve information about the example Pod's network interface:
+
+```shell
+ip link show type veth up
+```
+
+The output is similar to:
+
+```
+6: veth85d4e050@if2: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP mode DEFAULT group default
+...
+```
+
+Use the output to retrieve the IPv4 address of the `veth` interface:
+
+```shell
+ip -f inet address show dev veth85d4e050
+```
+
+The output is similar to:
+
+```
+    inet 10.241.1.1/32 scope global veth5a017fc1
+    ...
+```
+
+You can query the Kubelet's API endpoint at `http://localhost:10250/pods`:
+
+```shell
+curl http://localhost:10250/pods
+```
+
+Tip: Use `jq` to "prettify" the JSON output:
+
+```shell
+curl http://10.241.1.1:10250/pods | jq '.'
```

Comment:
```suggestion
curl http://localhost:10250/pods | jq '.'
```

Reply:
Done

Reply:
@sftim I think I made a mistake (HEAD number) when I squashed the commits and accidentally included doc `content/es/docs/tasks/network/_index.md` in my last commit, That doc is from another PR. How can I fix it?

Reply:
@sftim, I think I've managed to un-squash the PR and now I don't see that network file.

Reply:
I think that the **k8s-ci-robot** added the `area/localization` and `language/es` labels because of that network file. Sorry!